<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>716</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="consoleTextWidget" name="consoleWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>1191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1191</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Прямое управление</string>
     </attribute>
     <widget class="directManageWidget" name="DMWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Сверление</string>
     </attribute>
     <widget class="drillWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1171</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Рисование</string>
     </attribute>
     <widget class="drawWidget" name="dWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1191</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Связь</string>
    </property>
    <addaction name="conntcnAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="conntcnAction">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>directManageWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/directmanagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>consoleTextWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/consoletextwidget.h</header>
  </customwidget>
  <customwidget>
   <class>drawWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/drawwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>drillWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/drillwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
